prov/efa/configure.m4

```diff
@@ -113,6 +114,46 @@ AC_DEFUN([FI_EFA_CONFIGURE],[
 	AC_SUBST(efa_CPPFLAGS)
 	AC_SUBST(efa_LDFLAGS)
 	AC_SUBST(efa_LIBS)
+
+	AC_ARG_ENABLE([efa-unit-test],
+		[AS_HELP_STRING([--enable-efa-unit-test=CMOCKA_INSTALL_DIR],
+				[Provide a path to the CMocka installation directory
+				 in order to enable EFA Unit Tests.])])
+
+	cmocka_dir=""
+	if [test x"$enable_efa_unit_test" = x"yes"]; then
+    	cmocka_dir=""
+	else
```

Comment:
Can `enable_efa_unit_test` be `no`?

Reply:
enable_efa_unit_test can be no if you pass --disable-efa-unit-test, updated if check to handle it to:

```
if [ test x"$enable_efa_unit_test" != x"" && test x"$enable_efa_unit_test" != x"no" ];
```